src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/dto/IncentiveReviewSummary.kt

```diff
@@ -30,8 +30,8 @@ data class IepSummary(
   val iepTime: LocalDateTime,
   @Schema(description = "Location  of prisoner when review took place within prison (i.e. their cell)", example = "1-2-003", required = false)
   val locationId: String? = null,
-  @Schema(description = "IEP Review History (descending in time)", required = true)
-  var iepDetails: List<IepDetail>,
+  @Schema(description = "Incentive Review History (descending in time)", required = true)
+  var incentiveReviewDetails: List<IncentiveReviewDetail>,
```

Comment:
changing the DTO’s field name here changes all existing apis, doesn’t it? i don’t think we want to do that…
(i like the improved description though)

Reply:
corrected